src/probnum/randvars/_scipy_stats.py

```diff
@@ -221,11 +219,18 @@ def in_support(x):
     else:
         in_support = None
 
+    def sample_from_scipy_rv(rng, size):
+        return scipy_rv.rvs(size=size, random_state=rng)
+
+    if hasattr(scipy_rv, "rvs"):
+        sample_wrapper = sample_from_scipy_rv
+    else:
+        sample_wrapper = None
```

Comment:
@marvinpfoertner I am not sure I understood this block of code perfectly. My goal here was to squeeze the scipy sampler into the probnum randomvariable sample interface (with input rng & size). If there is a more elegant way of doing this, please let me know.